reference/conanfile/methods/configure.rst

```diff
@@ -40,10 +40,10 @@ so it should be removed:
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
```

Comment:
I am using the code that we are using so far in all ``conan new`` templates. If ``rm_safe`` is better solution for some reason, we should probably change the templates. Meanwhile we probably want to leave the same in the docs?

Reply:
Is something specific to `fPIC`.
For what I've seen in CCI, it's normal for projects to both define `config_options` where they `del self.options.fPIC` if in Windows, and to also define `configure` where they also delete `fPIC`, but if shared, just like in this example.

That naturally fails in Windows as a double delete happens (https://github.com/conan-io/conan-center-index/pull/15247 for an example where I had to fix it)

My suggestion would be to prefer to use `rm_safe` just so that is less likely to happen - maybe modifying the templates is also the way to go :)

Reply:
Ok, agree, but I'd prefer to adjust the templates first, then align the docs.
This shared-fPIC issue is actually a 2.X one (https://github.com/conan-io/conan/issues/12121), and there are also a couple of open tickets for 2.0 too:
- https://github.com/conan-io/conan/issues/8375
- https://github.com/conan-io/conan/issues/7530

Let's have a look to this separately.

Reply:
👍